src/actions/space.js

```diff
@@ -246,14 +244,14 @@ const saveSpace = async ({ space }) => async dispatch => {
   }
 };
 
-const clearSpace = () => dispatch => {
+const clearSpace = () => (dispatch) => {
   dispatch(clearPhase());
   return dispatch({
     type: CLEAR_SPACE,
   });
 };
 
-const deleteSpace = ({ id }) => dispatch => {
+const deleteSpace = ({ id }, onSuccess) => (dispatch) => {
```

Comment:
Why don't you put `onSuccess` inside the object?

Reply:
I prefer to keep the object and the callback apart. That way the payload for the back-end won't include onSuccess as well (though it doesn't change much).

Reply:
OK.